Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -336,6 +345,36 @@ Flexform settings can be read from within a Fluid template using
 :html:`{settings}`.
 
 
+.. _subtypes_excludelist:
+
+How to remove extra fields from FlexForm
+----------------------------------------
+
+In :file:`Configuration/TCA/Overrides/tt_content.php` add for example
+the following::
+
+   $GLOBALS['TCA']['tt_content']['types']['list']['subtypes_excludelist']['example_registration'] = 'recursive,pages';
+
+:ref:`t3tca:types-properties-subtypes-excludelist` excludes the fields
+*recursive* and *pages* from the FlexForm view. This is not strictly
+necessary, but declutters your FlexForm in the cases that these fields
+do not need to be configured. You should do this, unless you need these
+fields in your FlexForm.
```

Comment:
Technically these two fields are not related to FlexForms at all but are displayed for plugins in general. So this should be mentioned in a different context IMO.

Reply:
Ok. I will remove. 

Not sure if or where this information should go (TCA, chapter on "content elements" and "plugins"). I am saving it in an issue: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/issues/290